src/handler/session_store.rs

```diff
@@ -28,9 +28,13 @@ impl TryFrom<&Session> for session::Model {
     fn try_from(session: &Session) -> Result<Self, Self::Error> {
         Ok(Self {
             id: session.id().to_string(),
+            // unwrap safety: session object comes from the database and will be a valid
```

Comment:
nit: This time comes from the session handler, as we are converting into database format here